Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/dashboard/DashboardViewResolverImpl.java

```diff
@@ -143,7 +144,7 @@ public void handleClose(CloseEvent event, List<DashboardPanel> panels) {
         String id = event.getComponent().getId();
 
         for (DashboardPanel panel : panels) {
-            if (panel.getType().replace(" ", "_").toLowerCase().equals(id)) {
+            if (panel.getType().replace(" ", "_").toLowerCase(Locale.ENGLISH).equals(id)) {
```

Comment:
Why can't we use id.equalIgnoreCase check instead of the other way around? This option is more flexible than hard code English locale